packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterSignUpRequest.kt

```diff
@@ -73,20 +72,21 @@ data class FlutterSignUpRequest(val map: HashMap<String, *>) {
     }
 
     companion object {
-        fun validate(req : HashMap<String, *>?): Boolean {
+        fun validate(req : HashMap<String, *>?) {
+            val validationErrorMessage: String = "SignUp Request malformed."
+
             if (req == null) {
-                return false;
+                throw AmplifyException(validationErrorMessage, "Request map is null or malformed. Check that request is present and properly formed.")
```

Comment:
Should we be putting all these error messages into a separate constants file instead of having these "naked" strings in our code?

Reply:
Yeah - I'll work on this.